swift-sdk.xcodeproj/project.pbxproj

```diff
@@ -849,7 +858,7 @@
 			path = "host-app";
 			sourceTree = "<group>";
 		};
-		ACFCA72920EB02DB00BFB277 /* Tests */ = {
```

Comment:
We will need to change `Tests` to `tests` codecov.yml where we ignore this directory.